Sources/SwiftFormatRules/NeverUseForceTry.swift

```diff
@@ -38,6 +38,7 @@ public struct NeverUseForceTry: SyntaxLintRule {
   }
 
   public func visit(_ node: TryExprSyntax) -> SyntaxVisitorContinueKind {
+    guard !context.isRuleDisabled(self.ruleName, node: node) else { return .visitChildren }
```

Comment:
Can you put a TODO here that calls out future work to generalize this so that each rule/visit-method needs to manually do this check?

Reply:
Done.